src/test/php/org/bovigo/vfs/vfsStreamFileTestCase.php

```diff
@@ -129,6 +129,8 @@ public function seekEmptyFile()
         $this->assertEquals(0, $this->file->getBytesRead());
         $this->assertTrue($this->file->seek(2, SEEK_END));
         $this->assertEquals(2, $this->file->getBytesRead());
+        $this->assertFalse($this->file->seek(-1, SEEK_SET),'Seek before beginning of file');
+        $this->assertEquals(2, $this->file->getBytesRead());
```

Comment:
I know the existing test cases don't follow the "roughly one assertion per test" rule, but I really think new tests should do this. Otherwise it is harder to see what broke when a test fails. In this case two new test methods should be added, one for testing against the empty file and one against a non-empty file.